include/quicly/loss.h

```diff
@@ -235,10 +245,12 @@ inline int quicly_loss_on_alarm(quicly_loss_t *r, uint64_t largest_sent, uint64_
         *restrict_sending = 0;
         return quicly_loss_detect_loss(r, largest_acked, do_detect);
     }
-    /* PTO. Send at least and at most 2 packets. */
+    /* PTO. Send at least and at most 1 packet during early (and aggressive) probing and 2 packets otherwise. */
     ++r->pto_count;
```

Comment:
Do we need to set `r->pto_count` to `QUICLY_AGRESSIVE_PROBE_THRESHOLD` if it was zero and if there is some pending data to be sent?

Reply:
My bad, the design always increments `pto_count` from one, but applies different scale factors based on if there's something to send.